README.md

```diff
@@ -11,19 +11,50 @@ It might also works with other content in the OpenZIM format: http://www.openzim
 
 If your Internet access is expensive/rare/slow/unreliable/watched/censored, you still can browse this amazing amount of knowledge and culture.
```

Comment:
I know this text isn't knew, but instead of "this amazing amount of knowledge and culture" I suggest "this amazing repository of knowledge and culture"